src/stories/Library/material-search/material-search.scss

```diff
@@ -13,6 +13,9 @@ $_box-shadow: 0px 0px 0px 2px rgba(0, 0, 0, 0.25);
 $_loading-spinner-size: 40px;
 $_highlighted-item-bg-color: $_gin-header-bg-color;
 
+$_box_shadow_header: 0 4px 2px -2px rgba(0, 0, 0, 0.171);
+$_alert_background_header: #ecc8c577;
```

Comment:
Consider using [red/danger colors from Gin](https://git.drupalcode.org/project/gin/-/blob/8.x-3.x/styles/theme/variables.scss?ref_type=heads#L19-23) to make the element feel more native.

You have used a similar approach elsewhere. How about doing the same here.

Reply:
Yea agreed. Have changed it to use some gin colors.